docs-new/docs/graphql/core/guides/data-modelling/one-to-one.mdx

```diff
@@ -114,99 +118,89 @@ We can now:
+          "owner": {
+            "id": 1,
+            "name": "Justin"
           }
         },
         {
-          &quot;id&quot;: 2,
-          &quot;passport_number&quot;: &quot;F0004586&quot;,
-          &quot;owner&quot;: {
-            &quot;id&quot;: 2,
-            &quot;name&quot;: &quot;Beltran&quot;
+          "id": 2,
+          "passport_number": "F0004586",
+          "owner": {
+            "id": 2,
+            "name": "Beltran"
           }
         }
       ]
     }
-    }">
-
-    </div>
+  }`}
+/>
 
-## Insert using one-to-one relationships
+## Insert using one-to-one relationships {#one-to-one-insert}
 
 We can now:
 
 -   insert `passport_info` with their `owner` where the `owner` might
     already exist (assume unique `name` for `owner`):
 
-<div class="graphiql" view_only=""
-query="mutation upsertPassportInfoWithOwner {
-  insert_passport_info(objects: [
-    {
-      passport_number: &quot;X98973765&quot;,
-      owner: {
-        data: {
-          name: &quot;Kelly&quot;
-        },
-        on_conflict: {
-          constraint: owner_name_key,
-          update_columns: [name]
-        }
-      },
-    }
-  ]) {
-    returning {
-      passport_number
-      owner {
-        name
-      }
-    }
-  }
-}
-:response:
-{
-  &quot;data&quot;: {
-    &quot;insert_passport_info&quot;: {
-      &quot;returning&quot;: [
+<GraphiQLIDE
+  query={`mutation upsertPassportInfoWithOwner {
+      insert_passport_info(objects: [
         {
-          &quot;passport_number&quot;: &quot;X98973765&quot;,
-          &quot;owner&quot;: {
-            &quot;name&quot;: &quot;Kelly&quot;
+          passport_number: "X98973765",
+          owner: {
+            data: {
+              name: "Kelly"
+            },
+            on_conflict: {
+              constraint: owner_name_key,
+              update_columns: [name]
+            }
+          },
+        }
+      ]) {
+        returning {
+          passport_number
+          owner {
+            name
           }
         }
-      ]
+      }
+  }`}
+  response={`{
+    "data": {
+      "insert_passport_info": {
+        "returning": [
+          {
+            "passport_number": "X98973765",
+            "owner": {
+              "name": "Kelly"
+            }
+          }
+        ]
+      }
     }
-  }
-}">
-
-</div>
-
-<div class="note">
-
-<div class="title">
-
-Note
+  }`}
+/>
 
-</div>
+:::info Note
 
 You can avoid the `on_conflict` clause if you will never have conflicts.
 
-</div>
+:::
 
 ### Caveat for nested inserts
 
 Due to the way nested inserts are typically handled (described
-`here <pg_nested_inserts>`), the order of object insertion needs to be
-specified using the `insertion_order <ObjRelUsingManualMapping>` option
+[here](https://hasura.io/docs/latest/graphql/core/databases/postgres/mutations/insert.mdx)), the order of object insertion needs to be
+specified using the [insertion_order](/graphql/core/api-reference/syntax-defs.mdx) option
```

Comment:
incorrect linking - link should take to a subsection but taking to top of page

Reply:
`insertion_order` link is not fixed - missing hash-param

`#objrelusingmanualmapping`